src/kernel/TurbKineticEnergySSTTAMSSrcElemKernel.C

```diff
@@ -58,19 +58,18 @@ TurbKineticEnergySSTTAMSSrcElemKernel<AlgTraits>::
   dataPreReqs.add_gathered_nodal_field(sdrNp1_, 1);
   dataPreReqs.add_gathered_nodal_field(densityNp1_, 1);
   dataPreReqs.add_gathered_nodal_field(tvisc_, 1);
-  dataPreReqs.add_gathered_nodal_field(prod_, 1);
   dataPreReqs.add_gathered_nodal_field(alpha_, 1);
+  dataPreReqs.add_gathered_nodal_field(prod_, 1);
   dataPreReqs.add_master_element_call(SCV_VOLUME, CURRENT_COORDINATES);
   if (shiftedGradOp_)
     dataPreReqs.add_master_element_call(
       SCV_SHIFTED_GRAD_OP, CURRENT_COORDINATES);
   else
     dataPreReqs.add_master_element_call(SCV_GRAD_OP, CURRENT_COORDINATES);
 
+  dataPreReqs.add_master_element_call(SCV_SHAPE_FCN, CURRENT_COORDINATES);
```

Comment:
Why is this done this way? You're going to populate `SCV_SHAPE_FCN` even if you're not going to use it in `execute`?

Reply:
Fixed by making it more consistent with the other kernels.